cmd/thanos/rule.go

```diff
@@ -40,8 +40,8 @@ import (
 	"github.com/thanos-io/thanos/pkg/prober"
 	"github.com/thanos-io/thanos/pkg/promclient"
 	"github.com/thanos-io/thanos/pkg/query"
-	thanosrule "github.com/thanos-io/thanos/pkg/rule"
-	v1 "github.com/thanos-io/thanos/pkg/rule/api"
+	v1 "github.com/thanos-io/thanos/pkg/rules/api"
+	thanosmanager "github.com/thanos-io/thanos/pkg/rules/manager"
```

Comment:
nit: maybe let's call the alias simply `rulesmanager` as `thanosmanager.Manager` as the context is lost.

Reply:
good point